### ISSUE TITLE
Not emitting es modules

### ISSUE THREAD

**Issue by jtmthf** [leebenson/reactql#8]
Very useful project here. One concern I had is that in `kit/webpack/browser.js`, `modules` is not set to `false` which means by default `import` statements are transpiled to `commonjs` meaning that not all tree shaking optimizations can be taken advantage of. However when I change this, there is no change in size of the output bundle. Not sure exactly why this is, and was wondering if you had an idea.

**Comment by leebenson** [comment]
Thanks @jtmthf, this looks like a bug. I'll work on this today.

**Comment by leebenson** [comment]
fixed in [1.4.1](https://github.com/leebenson/reactql/releases/tag/1.4.1).

Demo:

**test.js**
```js
export const A = 'this should show up';
export const B = 'this should never wind up in the bundle';
```

**app.js**
```js
import React from 'react';
import { A } from './test';

export default () => (
  <div>
    {A}
  </div>
);
```

**Resulting `dist/public/browser.js` bundle**:
```js
webpackJsonp([0],{124:function(e,n,t){"use strict";function r(){c.a.render(u.a.createElement(E,null),document.getElementById("main"))}Object.defineProperty(n,"__esModule",{value:!0});var i=t(172),o=(t.n(i),t(5)),u=t.n(o),a=t(187),c=t.n(a),l=t(262),s=t(91),d=(t.n(s),t(141)),f=t(142),_=t(143),p=t.i(d.a)(),v=t.i(f.a)(p),E=function(){var e=function(){return u.a.createElement(s.ApolloProvider,{store:v,client:p},u.a.createElement(l.a,null,u.a.createElement(_.a,null)))};return e}();r()},140:function(e,n,t){"use strict";t.d(n,"a",function(){return r});var r={uri:"https://api.graph.cool/simple/v1/cinomw2r1018601o42x5z69uc"}},141:function(e,n,t){"use strict";function r(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{};return new u.ApolloClient(Object.assign({reduxRootSelector:function(e){return e.apollo},networkInterface:c},e))}function i(){return r()}var o=t(5),u=(t.n(o),t(91)),a=(t.n(u),t(140));n.a=i;var c=t.i(u.createNetworkInterface)({uri:a.a.uri})},142:function(e,n,t){"use strict";function r(e){return t.i(i.createStore)(t.i(i.combineReducers)({apollo:e.reducer()}),window.__STATE__,t.i(i.compose)(t.i(i.applyMiddleware)(e.middleware()),void 0!==window.__REDUX_DEVTOOLS_EXTENSION__?window.__REDUX_DEVTOOLS_EXTENSION__():function(e){return e}))}var i=t(75);n.a=r},143:function(e,n,t){"use strict";var r=t(5),i=t.n(r),o=t(144);n.a=function(){return i.a.createElement("div",null,o.a)}},144:function(e,n,t){"use strict";t.d(n,"a",function(){return r});var r="this should show up"},291:function(e,n,t){e.exports=t(124)}},[291]);% 
```

> No mention of `const B...`
